cypress/integration/Train/MyNameIs.spec.js

```diff
@@ -30,7 +30,10 @@ describe('Train', () => {
     cy.WaitForTrainingStatusCompleted()
 
     train.TypeYourMessage('My name is Susan.')
+// TODO: Remove these left indented lines once the product bug is fixed.    
+cy.log('Going to hit bug 1948')
```

Comment:
Is this not fixed by adding the extra input into the dialog?  Shahin said he tested it yesterday, I thought it fixed it.

Reply:
I will try that, haven't tried yet...if I can get it working I'll remove the log and comment.

Reply:
Shahin did fix it as long as I add the extra input. I'll fix that in the test code in my next go at it. Need to play around with it a bit more before I decide how I want to test this and if indeed something else was not at issue here that might have been missed.